kolibri/plugins/learn/assets/src/views/content-page/index.vue

```diff
@@ -212,6 +216,9 @@
           this.content.kind === ContentNodeKinds.AUDIO
         );
       },
+      downloadableFiles() {
+        return this.content.files.filter(file => file.preset !== 'Thumbnail');
```

Comment:
At some point we will want to clean this up to shared frontend/backend constants.